couchpotato/core/media/_base/providers/torrent/morethantv.py

```diff
@@ -0,0 +1,132 @@
+import traceback
+
+from bs4 import BeautifulSoup
+from couchpotato.core.helpers.variable import tryInt
+from couchpotato.core.logger import CPLog
+from couchpotato.core.media._base.providers.torrent.base import TorrentProvider
+import six
+
+
+log = CPLog(__name__)
+
+
+class Base(TorrentProvider):
+
+    urls = {
+        'test': 'https://www.morethan.tv/',
+        'login': 'https://www.morethan.tv/login.php',
+        'login_check': 'https://www.morethan.tv/inbox.php',
+        'detail': 'https://www.morethan.tv/torrents.php?torrentid=%s',
+        'search': 'https://www.morethan.tv/torrents.php?filter_cat%%5B1%%5D=1&action=advanced&searchstr=%s',
+        'download': 'https://www.morethan.tv/%s',
+    }
+
+    http_time_between_calls = 1  # Seconds
+
+    def _search(self, media, quality, results):
```

Comment:
If you don't search on imdb url, please use `def _searchOnTitle(self, title, movie, quality, results)` this will loop over multiple title alternatives automatically. See bitsoup, ilovetorrents, iptorrents (and more) for examples.